assignments/this.js

```diff
@@ -1,10 +1,10 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Implicit binding: The most common of bindings and basically a referencing binding that is to the left of where the functin is called. 
+* 2. Explicit Binding: The ability to reference functions from outside the scope of the object. You're explicitly setting what the this keyword object is referencing. 
+* 3. New Binding: This is when you invoke the 'new' keyword and a 'new' this ojbect is created and bound to the new object being constructed. 
+* 4. Window Binding: The instance where you haven't used any of the binding rules, then by default the this keyword references the window object. 
```

Comment:
great answers to all of these. it should be noted that the Window binding happens specifically in the global scope, but as we talked about earlier this week, can also happen mistakenly when ES6 arrow functions are used.